### ISSUE TITLE
Local web server does not reflect any file changes

### ISSUE THREAD

**Issue by TimWerdin** [symfony-cli/symfony-cli#121]
Steps on how to reproduce:
1. Start web server `symfony server:start -d`
2. Load page in browser
3. Change some code in e.g. a controller or a twig extension so that the website should display something else like a "2" behind the title of the page
4. Reload the page in browser

What would you expect?
That the 2 is displayed behind the title

What is actually happening?
No change at all. I have to stop the web server and start it again to see the code change

Symfony cli version: 5.4.5
Symfony version: 5.3.*

**Comment by henrikac** [comment]
I just tried to generate a new 5.3.* project with `symfony new test --version=5.3`. Then I installed twig to get templates `composer require twig` and created `DefaultController::index` that just render `index.html.twig` which extends `base.html.twig`. Then I uncommented the comments in `config/routes.yaml` and navigated to `localhost:8000`.

The page looks as expected. I then updated the title in `base.html.twig` by appending `2` to the end of the title and pressed F5. I got the expected result - my title with the number 2 appended to the end.
